source/docs/appendices/migration-guides.md

```diff
@@ -50,6 +50,16 @@ In this version, we have improved the MondialRelay shipping support with Magento
 Here are some highlights of the main changes in upstream libraries that *unlikely may* impact your application. We have tested them and haven't found any regression, but we prefer to mention these changes in case you detect a weird issue after upgrading:
 - `axios` does not append the `charset=utf-8` anymore for requests with `Content-Type:application/json`. See [#680](https://gitlab.com/front-commerce/front-commerce/-/merge_requests/680#note_711807434), [#4016](https://github.com/axios/axios/issues/4016) and [#2154](https://github.com/axios/axios/issues/2154) and for details.
 
+## Unnecessary unsafeHtml for products and cart items
+
+Until now, the names of products and cart items needed to be binded as HTML content in the react components.
```

Comment:
That's not super actionable for the developer upgrading a FC project. Actually, you fixed 2 issues:

1. for cart item name, HTML entities were not decoded and they could appear on the site
1. for product name, HTML entities were decoded but a React component responsible for displaying was still using the `safeHtml` trick

for the 1, in terms of upgrade, there's nothing special to do unless, they use the `safeHtml` trick on their own when displaying it.

for the 2, the thing to do is to check if they have an override of the component and if so to remove the `safeHtml` trick